src/TrafficSign/TrafficSignHelper.ts

```diff
@@ -0,0 +1,347 @@
+import steepAscent15 from "./svg/steepAscent15.svg";
+import steepAscent16 from "./svg/steepAscent16.svg";
+import steepAscent17 from "./svg/steepAscent17.svg";
+import steepAscent18 from "./svg/steepAscent18.svg";
+import steepAscent19 from "./svg/steepAscent19.svg";
+import steepAscent20 from "./svg/steepAscent20.svg";
+import steepAscent21 from "./svg/steepAscent21.svg";
+import steepAscent22 from "./svg/steepAscent22.svg";
+import steepAscent23 from "./svg/steepAscent23.svg";
+import steepAscent24 from "./svg/steepAscent24.svg";
+import steepAscent25 from "./svg/steepAscent25.svg";
+import steepAscent4 from "./svg/steepAscent4.svg";
+import steepAscent5 from "./svg/steepAscent5.svg";
+import steepAscent6 from "./svg/steepAscent6.svg";
+import steepAscent7 from "./svg/steepAscent7.svg";
+import steepAscent8 from "./svg/steepAscent8.svg";
+import steepAscent9 from "./svg/steepAscent9.svg";
+import steepDescent10 from "./svg/steepDescent10.svg";
+import steepDescent11 from "./svg/steepDescent11.svg";
+import steepDescent12 from "./svg/steepDescent12.svg";
+import steepDescent13 from "./svg/steepDescent13.svg";
+import steepDescent14 from "./svg/steepDescent14.svg";
+import steepDescent15 from "./svg/steepDescent15.svg";
+import steepDescent16 from "./svg/steepDescent16.svg";
+import steepDescent17 from "./svg/steepDescent17.svg";
+import steepDescent18 from "./svg/steepDescent18.svg";
+import steepDescent19 from "./svg/steepDescent19.svg";
+import steepDescent20 from "./svg/steepDescent20.svg";
+import steepDescent21 from "./svg/steepDescent21.svg";
+import steepDescent22 from "./svg/steepDescent22.svg";
+import steepDescent23 from "./svg/steepDescent23.svg";
+import steepDescent24 from "./svg/steepDescent24.svg";
+import steepDescent25 from "./svg/steepDescent25.svg";
+import steepDescent4 from "./svg/steepDescent4.svg";
+import steepDescent5 from "./svg/steepDescent5.svg";
+import steepDescent6 from "./svg/steepDescent6.svg";
+import steepDescent7 from "./svg/steepDescent7.svg";
+import steepDescent8 from "./svg/steepDescent8.svg";
+import steepDescent9 from "./svg/steepDescent9.svg";
+import stop from "./svg/stop.svg";
+import straightOrLeft from "./svg/straightOrLeft.svg";
+import straightOrRight from "./svg/straightOrRight.svg";
+import trafficLight from "./svg/trafficLight.svg";
+import trafficQueues from "./svg/trafficQueues.svg";
+import tunnel from "./svg/tunnel.svg";
+import turnAheadL from "./svg/turnAheadL.svg";
+import turnAheadR from "./svg/turnAheadR.svg";
+import turnL from "./svg/turnL.svg";
+import turnR from "./svg/turnR.svg";
+import unevenRoad from "./svg/unevenRoad.svg";
+import urbanArea from "./svg/urbanArea.svg";
+import urbanAreaEnd from "./svg/urbanAreaEnd.svg";
+import warningPedestrianCrossing from "./svg/warningPedestrianCrossing.svg";
+import weightLimit from "./svg/weightLimit.svg";
+import widthLimit from "./svg/widthLimit.svg";
+
+interface IRoadSignsObject {
```

Comment:
type rather than interface should be the default unless you need any features of interface

Reply:
Fixed